paddle/phi/backends/onednn/onednn_reuse.h

```diff
@@ -1074,6 +1074,68 @@ class BroadcastDataOneDNNHandler
   }
 };
 
+template <typename T>
+class PReluOneDNNHandler
+    : public OneDNNHandlerNoCachingT<T,
+                                     dnnl::prelu_forward,
+                                     dnnl::prelu_backward> {
+ public:
+  PReluOneDNNHandler(const OneDNNContext& dev_ctx,
+                     const dnnl::engine engine,
```

Comment:
```suggestion
  PReluOneDNNHandler(const dnnl::engine engine,
```
No need to pass whole `OneDNNContext` to handler, as you already extracted `engine`, `place` and `is_test` in both kernels

Reply:
Do you think I should also make `engine` and `cpu_place` as `const &`?